client.py

```diff
@@ -1,95 +1,130 @@
-import hazelcast
 import logging
 import random
+
+import hazelcast
 from hazelcast.core import HazelcastJsonValue
 from hazelcast.discovery import HazelcastCloudDiscovery
 
 """
 This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
 After successful connection, it puts random entries into the map.
 
-See: https://docs.hazelcast.cloud/docs/python-client
+See: https://docs.hazelcast.com/cloud/python-client
 """
 
 
-def map_example(my_map):
-    print("Now, 'map' will be filled with random entries.")
+def city(country: str, name: str, population: int) -> HazelcastJsonValue:
+    return HazelcastJsonValue({
+        "country": country,
+        "city": name,
+        "population": population,
+    })
 
-    iteration_counter = 0
-    while True:
-        random_key = str(random.randint(1, 100000))
-        my_map.put("key" + random_key, "value" + random_key)
-        my_map.get("key" + random_key)
-        iteration_counter += 1
-        if iteration_counter == 10:
-            iteration_counter = 0
-            print("Map size:", my_map.size())
+
+def country(code: str, name: str) -> HazelcastJsonValue:
+    return HazelcastJsonValue({
+        "isoCode": code,
+        "country": name,
+    })
 
 
-def sql_example(hz_client):
+def map_example(client: hazelcast.HazelcastClient):
+    """
+    This example shows how to work with Hazelcast maps.
+    """
+    cities = client.get_map("cities").blocking()
+    cities.put("1", city("United Kingdom", "London", 9_540_576))
+    cities.put("2", city("United Kingdom", "Manchester", 2_770_434))
+    cities.put("3", city("United States", "New York", 19_223_191))
+    cities.put("4", city("United States", "Los Angeles", 3_985_520))
+    cities.put("5", city("Turkey", "Ankara", 5_309_690))
+    cities.put("6", city("Turkey", "Istanbul", 15_636_243))
+    cities.put("7", city("Brazil", "Sao Paulo", 22_429_800))
+    cities.put("8", city("Brazil", "Rio de Janeiro", 13_634_274))
+    map_size = cities.size()
+    print(f"'cities' map now contains {map_size} entries.")
+    print("-"*20)
+
+
+def sql_example(client: hazelcast.HazelcastClient):
+    svc = client.sql
```

Comment:
Can we rename `svc` to `sql_service` to be consistent with the parameter names?

Reply:
Updated at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4